app/views/questions/show.html.erb

```diff
@@ -0,0 +1,53 @@
+
+<p>
+  <h1>
+  	<strong>Title:</strong>
+  	<%= @question.title %>
+  </h1>
+</p>
+
+<p>
+  <!-- <strong>Description:</strong> -->	
+  <div class="question_description"> <%= @question.description %> </div>
+</p>
+
+<% if can? :update, @question %>
+		<%= link_to 'Edit', edit_question_path(@question) %> 
+		<%= link_to 'New Question', new_question_path %>
+<% end %>
+
+<div class="comments" name="comments" >
+	<h2> Comments </h2>
+	<%= render 'comments/show_selective', comments: @comments  %>
+	<br>
+	<%= render 'comments/form', comment: @comment  %>
+</div>
+
+<br>
+<div class="answers" name="answers">
+	<h2> Answers </h2>
+	
+	<div name="answers_list" >
+		<% @answers.each do |answer| %>
+			<h3> Answer </h3>
+			<% if answer.user.present?%>
```

Comment:
is there any case that an answer is given by some unknown user? if not then you need to remove this check.